### DISCUSSION TITLE
Texstudio 4.4.1 and Macros Scripts

### DISCUSSION THREAD

**Discussion by ldman87** [texstudio-org/texstudio#2888] [Q&A] [answered]
%SCRIPT
app.fileNew();
newEditor = documentManager.currentDocument.editorView.editor;
newEditor.write("Hello!");

Don't work at texstudio 4.4.1 ?

**Comment by sunderme** [comment, marked as answer]
that was a bug, fixed for next release

**Reply by ldman87** [reply]
Thank you very much!

**Reply by ldman87** [reply]
Macro:   editorView.editor.write("hello!");   doesn't execute correctly.
